.github/workflows/dashboard-ci.yaml

```
@@ -0,0 +1,107 @@
+name: Token Dashboard / CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  format:
+    name: Check code format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Check formatting
+        run: yarn format
+
+  build-and-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build
+        run: yarn build
+
+      # FIXME: Tests are missing, will this step be needed?
+      # - name: Test
+      #   run: yarn test
+
+  build-and-deploy-testnet:
+    name: Deploy to testnet
+    needs: build-and-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      # FIXME: It's work in progress, the contracts are not yet published.
+      # - name: Resolve latest ropsten contracts
+      #   run: |
+      #     yarn upgrade \
+      #       @threshold-network/contracts@ropsten
+
+      - name: Build
+        run: yarn build
+
+      # A pull_request event is a PR; deploy to preview testnet bucket.
+      - name: Deploy PR preview to GCP
+        if: github.event_name == 'pull_request'
+        uses: thesis/gcp-storage-bucket-action@v3.1.0
+        with:
+          service-key: ${{ secrets.KEEP_TEST_CI_UPLOAD_DAPP_JSON_KEY_BASE64 }}
+          project: ${{ secrets.KEEP_TEST_GOOGLE_PROJECT_ID }}
+          bucket-name: preview.dashboard.test.threshold.network
+          bucket-path: ${{ github.head_ref }}
+          build-folder: build
+
+      - name: Post preview URL to PR
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v3
+        with:
+          script: |
+            github.issues.createComment({
```

We need to call the `issues.createComment` command using `github.rest.` context (["the Octokit context available via github no longer has REST methods directly"](https://github.com/actions/github-script#breaking-changes-in-v5)). Also, we need to use `v5` of the `github-script` action, which supports the new changes.

```suggestion
        uses: actions/github-script@v5
        with:
          script: |
            github.rest.issues.createComment({
```

FYI, I'm going to apply same change to the https://github.com/keep-network/keep-core/blob/main/.github/workflows/dashboard-mainnet.yml workflow, which also uses the `github-script` action.